salt/states/virt.py

```diff
@@ -265,7 +265,10 @@ def running(name,
             password=None,
             os_type=None,
             arch=None,
-            boot=None):
+            boot=None,
+            serial_type=None,
+            console=True,
+            telnet_port=None):
```

Comment:
Naming it just `port` is more appropriate imho

Reply:
This is the same name being used by the virt execution module. It's probably a good idea to keep the naming syntax the same for compatibility sake.

Reply:
Oh ok, 100% agree

Reply:
well, ideally I would even deprecate those parameters to go for a more flexible version of it to help covering more of the libvirt definitions. This implementation has already been started in PR #56430 and it would be nice to refresh and finish that PR